src/Weasel.Postgresql/Weasel.Postgresql.csproj

```diff
@@ -18,7 +18,7 @@
 
     <ItemGroup>
       
-      <PackageReference Include="Npgsql" Version="5.0.7" />
+      <PackageReference Include="Npgsql" Version="6.0.0-rc.1" />
```

Comment:
they have released v6.0.0 already

Reply:
Thank you for giving the heads up. We're aware of the release. However, it has many breaking changes in the core parts that Marten/Weasel depends on. We're discussing with the Npgsql team if we could find some solution. If not, then we'll have to workaround that.